Source/missiles.cpp

```diff
@@ -52,12 +52,8 @@ void GetDamageAmt(int i, int *mind, int *maxd)
 			*mind += *mind / 2;
 		}
 		*maxd = 10;
-		for (int k = 0; k < myPlayer._pLevel; k++) {
-			*maxd += 4;
-		}
-		for (int k = 0; k < sl; k++) {
-			*maxd += 6;
-		}
+		*maxd += 4 * myPlayer._pLevel;

```

Comment:
These changes come from your other PR, right? Did you mean to include these here?

Reply:
fixed